.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Extractors/ChineseMergedExtractorConfiguration.cs

```diff
@@ -59,6 +63,8 @@ public List<ExtractResult> Extract(string text, DateObject referenceTime)
 
             CheckBlackList(ref ret, text);
```

Comment:
Why do we need to both CheckDenyList and then FilterAmbiguity? Shouldn't they be the same or at least the ambiguity filter be inside the other method?